functions/db/deliveryRequests/onUpdate.f.js

```diff
@@ -101,14 +104,23 @@ function sendRequestToDGs(accountsRef, requestSnap) {
         console.info('No specific DG requested, ' + pending.length + ' pending DGs');
     }
 
-    let promises = [];
-    for (let dg in pending) {
-        let dgRef = accountsRef.child(pending[dg]);
-        promises.push(
-            utils.notifyRequestUpdate(
-                'DG', dgRef, requestSnap.key, nt.RECURRING_PICKUP_REQUEST));
-    }
+    return Promise.all(pending.map((dgId) => {
+        return utils.notifyRequestUpdate(
+            'DG', accountsRef.child(dgId), requestSnap.key, nt.RECURRING_PICKUP_REQUEST);
+    }));
+}
```

Comment:
you can go one level cooler by removing the return if you want:
```
return Promise.all(pending.map((dgId) => utils.notifyRequestUpdate(...));
```

Reply:
😂😂 or even cooler, remove the bracket: `map( dgId => utils.notifyRequestUpdate(...) )`